Resources/Prototypes/Corvax/Entities/Objects/Misc/flatpacks.yml

```
@@ -0,0 +1,8 @@
+- type: entity
+  parent: BaseFlatpack
+  id: PrinterDocFlatpack
+  name: document printer flatpack
+  description: A flatpack used for constructing a document printer.
+  components:
+  - type: Flatpack
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)